### ISSUE TITLE
template dokuwiki - js.php - access is denied

### ISSUE THREAD

**Issue by githubrota** [splitbrain/dokuwiki#1058]
When opening the wiki in a internet explorer browser dialog (internetexplorer window in another program) and using the template dokuwiki (and only then) following error is produced:
![scripterror](https://cloud.githubusercontent.com/assets/11191098/6369023/922d8670-bce2-11e4-8083-3806a2671f5a.png)

IE version 11
it does not matter what page is initally opened. (doku.php)
Release 2014-09-29b "Hrun"

the java script code at that position:
return c||(c=Eb(a,b),"none"!==c&&c||(Cb=(Cb||m("<iframe frameborder='0' width='0' height='0'/>")).appendTo(b.documentElement),b=(Cb[0].contentWindow||Cb[0].contentDocument).document,b.write(),b.close(),c=Eb(a,b),Cb.detach()),Db[a]=c),c}!function(){var a;k.shrinkWrapBlocks=function(){if(null!=a)return a;a=!1;var b,c,d;return c=y.getElementsByTagName("body")[0],c&&c.style?(b=y.createElement("div"),d=y.createElement("div"),d.style.cssText="position:absolute;border:0;width:0;height:0;top:0;left:-9999px",c.appendChild(d).appendChild(b),typeof b.style.zoom!==K&&(b.style.cssText="-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;display:block;margin:0;border:0;padding:1px;width:1px;zoom:1",b.appendChild(

the page is rendered correctly in the background but when canceling the error message it jumps to about:blank

**Comment by splitbrain** [comment]
Sorry, that's not enough detail. What IE version? What are you doing?

**Comment by githubrota** [comment]
sorry, post is still under construction. image upload was not working.

**Comment by githubrota** [comment]
in a "normal" internet explorer window it is working.

**Comment by githubrota** [comment]
if a browser dialog program is needed to try and reproduce the error, then i could provide that...

**Comment by githubrota** [comment]
allright after tinkering around i found out that the problem was generated by renavigation within the browser dialog.
